framerail/.gitattributes

```diff
@@ -0,0 +1,2 @@
+pnpm-lock.yaml linguist-generated=true
```

Comment:
pnpm lockfiles should be `binary linguist-vendored linguist-generated`

The generated TS file is fine as-is